app/build.gradle

```diff
@@ -142,7 +142,7 @@ def koin_version = "2.1.6"
 def okhttp_version = "4.8.1"
 def retrofit_version = "2.9.0"
 def room_version = "2.2.5"
-def suitcase_version = "5.2.0"
+def suitcase_version = "6.0.0"
```

Comment:
This can be v6.0.1 now, and will allow you to use the right `firebase-analytics` dependency below.